src/database/migrations/20200312122638-create-trip.js

```diff
@@ -0,0 +1,61 @@
+
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('trips', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    userId: {
+      allowNull: false,
+      type: Sequelize.INTEGER,
+      references: {
+        model: 'users',
+        key: 'id',
+      },
+    },
+    travelFrom: {
+      allowNull: false,
+      type: Sequelize.STRING
+    },
+    travelTo: {
+      allowNull: false,
+      type: Sequelize.STRING
+    },
+    travelDate: {
+      allowNull: false,
+      type: Sequelize.DATE
+    },
+    travelReason: {
+      allowNull: false,
+      type: Sequelize.STRING
+    },
+    travelType: {
+      allowNull: false,
+      type: Sequelize.STRING
+    },
+    isApproved: {
```

Comment:
The request status should not be a boolean since it can be pending, approved or rejected. if we use a boolean we will never know if it is still pending or if it has been rejected.